ui.css

```diff
@@ -133,15 +141,15 @@ button.reveal-death {
 }
 
 .chat {
-  position: fixed;
-  bottom: 0;
   width: 100%;
   backdrop-filter: blur(10px);
   background: rgba(255, 255, 255, 0.8);
   border: 1px solid #aaa;
   box-sizing: border-box;
   border-radius: 5px;
-  height: 30vh;
+  max-height: 30vh;
+  min-height: 120px;
```

Comment:
At 120px the chat can hold about 1 line of people talking. This probably suggests a need for more compact message structure (and removing the margin around the silly mdl textbox), but would make it pretty difficult to follow conversations.